tests/configuration.py

```diff
@@ -33,5 +34,5 @@
 docker.use_docker = bool(strtobool(os.getenv('USE_DOCKER', 'True')))
 
 # Printing all environment keys
-for key, value in sorted(os.environ.items()):
-    print(f'{key}: {value}')
+# for key, value in sorted(os.environ.items()):
+#     print(f'{key}: {value}')
```

Comment:
we should avoid commenting out code blocks like that, can be removed

Reply:
removed